[2025-01-02T06:35:45.171+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-02T06:35:45.210+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_transform_news_data.process_news_all_countries manual__2025-01-02T06:35:19.316340+00:00 [queued]>
[2025-01-02T06:35:45.231+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_transform_news_data.process_news_all_countries manual__2025-01-02T06:35:19.316340+00:00 [queued]>
[2025-01-02T06:35:45.233+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 3
[2025-01-02T06:35:45.259+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): process_news_all_countries> on 2025-01-02 06:35:19.316340+00:00
[2025-01-02T06:35:45.269+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=131) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-02T06:35:45.272+0000] {standard_task_runner.py:72} INFO - Started process 133 to run task
[2025-01-02T06:35:45.273+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_transform_news_data', 'process_news_all_countries', 'manual__2025-01-02T06:35:19.316340+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/fetch_and_transform_news.py', '--cfg-path', '/tmp/tmp8fwnv631']
[2025-01-02T06:35:45.276+0000] {standard_task_runner.py:105} INFO - Job 79: Subtask process_news_all_countries
[2025-01-02T06:35:45.358+0000] {task_command.py:467} INFO - Running <TaskInstance: fetch_and_transform_news_data.process_news_all_countries manual__2025-01-02T06:35:19.316340+00:00 [running]> on host 4fe538676cbc
[2025-01-02T06:35:45.502+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_transform_news_data' AIRFLOW_CTX_TASK_ID='process_news_all_countries' AIRFLOW_CTX_EXECUTION_DATE='2025-01-02T06:35:19.316340+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-02T06:35:19.316340+00:00'
[2025-01-02T06:35:45.505+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-02T06:35:45.545+0000] {base.py:84} INFO - Retrieving connection 'snowflake_con'
[2025-01-02T06:35:45.548+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.4, Python Version: 3.12.8, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-01-02T06:35:45.550+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2025-01-02T06:35:45.552+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-01-02T06:35:46.900+0000] {logging_mixin.py:190} INFO - Error fetching offset: 250001 (08001): None: Failed to connect to DB: qkjqutz-knb84971.snowflakecomputing.com:443. Incorrect username or password was specified.
[2025-01-02T06:35:46.902+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.4, Python Version: 3.12.8, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-01-02T06:35:46.904+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2025-01-02T06:35:46.905+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-01-02T06:35:48.124+0000] {logging_mixin.py:190} INFO - Error transforming data into Snowflake: 250001 (08001): None: Failed to connect to DB: qkjqutz-knb84971.snowflakecomputing.com:443. Incorrect username or password was specified.
[2025-01-02T06:35:48.125+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-02T06:35:48.143+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-02T06:35:48.145+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fetch_and_transform_news_data, task_id=process_news_all_countries, run_id=manual__2025-01-02T06:35:19.316340+00:00, execution_date=20250102T063519, start_date=20250102T063545, end_date=20250102T063548
[2025-01-02T06:35:48.225+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-02T06:35:48.257+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
